Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -127,18 +129,9 @@ public void setTasksStatusUp(Process process) throws DataException, IOException
      * @param process
      *            object
      */
-    public void setTasksStatusDown(Process process) throws DataException {
-        List<Task> tasks = new CopyOnWriteArrayList<>(process.getTasks());
-        Collections.reverse(tasks);
-
-        for (Task task : tasks) {
-            // TODO: check if this behaviour is correct
-            if (process.getTasks().get(0) != task && task.getProcessingStatus() != TaskStatus.LOCKED) {
-                setTaskStatusDown(task);
-                taskService.save(task);
-                break;
-            }
-        }
+    public void setTasksStatusDown(Process process) {
+        Task currentTask = ServiceManager.getProcessService().getCurrentTask(process);
```

Comment:
`getCurrentTask` should not be used anymore as there might be more than one current task per process.